src/inference/tests/functional/ov_extension_test.cpp

```diff
@@ -225,23 +225,6 @@ class CustomOldIdentity : public ngraph::op::Op {
     }
 };
 
-class TestTileOldExtension : public InferenceEngine::IExtension {
```

Comment:
Why do you remove? Looks like it should be migrated to new API

Reply:
These old api test are moved to legacy/ov_extension_test.cpp